### DISCUSSION TITLE
Little bar on top when stacking windows one on top of another

### DISCUSSION THREAD

**Discussion by MoutonBinoclard** [FancyWM/fancywm#204] [Q&A] [answered]
![image](https://github.com/FancyWM/fancywm/assets/77278792/edeb6cef-2354-4806-9ab7-5a4e274f44e4)
Hello everyone, when i put a window on top of another, i got this banner. I don't understand the need for it. Can someone explain me if I can get rid of it or just why is it important  ?
Thanks

**Comment by coffeeneed** [comment]
Same for me. I love FancyVM, but it just seems like another taskbar to me. Is it possible to disable it?

**Comment by veselink1** [comment, marked as answer]
The panel indicates what layout is applied to a group of windows.

Normally, the top level panel for a desktop is horizontal panel. The top level panel does not have this bar visible, but any other nested panels do. It is possible to nest a horizontal panel inside another horizontal panel, in which case, I expect you will see something like in the screenshot.
